File: Java/poo/salary/library/pension.java
package poo.salary.library;

public class pension {

    private double valuePension;

    public pension(double valuePension) {
        this.valuePension = valuePension;
    }

    public double getValuePension() {
        return this.valuePension;
    }

    public double calculateDiscount() {
        return (this.getValuePension() * 0.04);
    }
}

File: Php/poo/salary/library/pension.php
<?php

    class pension extends person {
    private $valuePension;
    
    public function __construct($valuePension) {
        $this->valuePension = $valuePension;
    }

    public function getValuePension(): float  {
        return $this->valuePension;
    }

    public function calculateDiscount(): float {
        $this->calculate = ($this->getValuePension() * 0.04);
        return $this->calculate;
    }
}